app/templates/install.hbs

```diff
@@ -4,36 +4,5 @@
 </div>
 
 <p>
-Rust comes with Cargo by default, so the easiest way to get Cargo is to
-<a href='http://www.rust-lang.org/install.html'>install Rust</a> via
-the official binary installers.
+    Redirecting you to <code>https://crates.io/</code>&#8230;
```

Comment:
Shouldn't this say "Redirecting you to `http://doc.crates/io`"? Since that's where the redirect is actually going...?

Reply:
Right. Updated.